### ISSUE TITLE
run micro server command error

### ISSUE THREAD

**Issue by xesia1989** [micro/micro#2055]
D:\micro>micro server

2024/01/24 15:15:04 [35m/home/runner/go/pkg/mod/gorm.io/gorm@v1.24.1-0.20221019064659-5dd2bb482755/gorm.go:206
[0m[31m[error] [0mfailed to initialize database, got error Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requi
res cgo to work. This is a stub

2024/01/24 15:15:04 [35m/home/runner/go/pkg/mod/gorm.io/gorm@v1.24.1-0.20221019064659-5dd2bb482755/gorm.go:206
[0m[31m[error] [0mfailed to initialize database, got error Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requi
res cgo to work. This is a stub
2024-01-24 15:15:04  file=server/server.go:74 level=info Starting server
2024-01-24 15:15:04  file=server/server.go:129 level=info Registering registry
2024-01-24 15:15:04  file=server/server.go:129 level=info Registering broker
2024-01-24 15:15:04  file=server/server.go:129 level=info Registering network
2024-01-24 15:15:04  file=server/server.go:129 level=info Registering runtime
2024-01-24 15:15:04  file=server/server.go:129 level=info Registering config
2024-01-24 15:15:04  file=server/server.go:129 level=info Registering store
2024-01-24 15:15:04  file=server/server.go:129 level=info Registering events
2024-01-24 15:15:04  file=server/server.go:129 level=info Registering auth
2024-01-24 15:15:04  file=server/server.go:139 level=info Starting runtime
2024-01-24 15:15:04  file=grpc/grpc.go:992 level=info Server [grpc] Listening on [::]:8081
2024-01-24 15:15:04  file=http/http.go:93 level=info HTTP Server Listening on [::]:8080
2024-01-24 15:15:04  file=local/service.go:162 level=info Service runtime has stopped with status: error
2024-01-24 15:15:04  file=local/service.go:173 level=info Restarting service runtime
2024-01-24 15:15:04  file=local/service.go:162 level=info Service runtime has stopped with status: error
2024-01-24 15:15:04  file=local/service.go:173 level=info Restarting service runtime
2024-01-24 15:15:05  file=local/service.go:162 level=info Service runtime has stopped with status: error

**Comment by asim** [comment]
Untested on windows. Might need to use power shell with root.

**Comment by HamedGhoochanian** [comment]
same thing happens in linux (ubuntu 22.04)
this is a fresh install using the official installer script


micro server

2024/01/28 13:05:59 /home/runner/go/pkg/mod/gorm.io/gorm@v1.24.1-0.20221019064659-5dd2bb482755/gorm.go:206
[error] failed to initialize database, got error Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work. This is a stub

2024/01/28 13:05:59 /home/runner/go/pkg/mod/gorm.io/gorm@v1.24.1-0.20221019064659-5dd2bb482755/gorm.go:206
[error] failed to initialize database, got error Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work. This is a stub
2024-01-28 13:05:59  file=server/server.go:74 level=info Starting server
2024-01-28 13:05:59  file=server/server.go:129 level=info Registering registry

**Comment by asim** [comment]
Recompile yourself?

**Comment by HamedGhoochanian** [comment]
Compiled it using the Makefile and it worked fine.

**Comment by alex-dna-tech** [comment]
Indirect dependency: [model -> gorm -> go-sqlite3](https://github.com/micro/micro/blob/1eb92cce4cb993441dc91338e2220af0972d538e/go.mod#L64)

https://github.com/mattn/go-sqlite3?tab=readme-ov-file#installation
> Important: because this is a CGO enabled package, you are required to set the environment variable CGO_ENABLED=1 and have a gcc compiler present within your path.

The [Docker version](https://github.com/micro/micro/blob/1eb92cce4cb993441dc91338e2220af0972d538e/Dockerfile#L11) works fine because it is built by [Makefile build target](https://github.com/micro/micro/blob/1eb92cce4cb993441dc91338e2220af0972d538e/Makefile#L24)

```shell-session
~$ docker run -it micro/micro:latest server
2024-03-31 08:22:59  file=user/user.go:99 level=info Setting up keys for JWT at /root/.micro/id_rsa and /root/.micro/id_rsa.pub
2024-03-31 08:23:01  file=server/server.go:74 level=info Starting server
2024-03-31 08:23:01  file=server/server.go:129 level=info Registering registry
2024-03-31 08:23:01  file=server/server.go:129 level=info Registering broker
2024-03-31 08:23:01  file=server/server.go:129 level=info Registering network
2024-03-31 08:23:01  file=server/server.go:129 level=info Registering runtime
2024-03-31 08:23:01  file=server/server.go:129 level=info Registering config
2024-03-31 08:23:01  file=server/server.go:129 level=info Registering store
2024-03-31 08:23:01  file=server/server.go:129 level=info Registering events
2024-03-31 08:23:01  file=server/server.go:129 level=info Registering auth
2024-03-31 08:23:01  file=server/server.go:139 level=info Starting runtime
2024-03-31 08:23:01  file=grpc/grpc.go:992 level=info Server [grpc] Listening on [::]:8081
2024-03-31 08:23:01  file=http/http.go:93 level=info HTTP Server Listening on [::]:8080
```

All [Release binaries](https://github.com/micro/micro/releases) are built by [goreleaser](https://github.com/micro/micro/blob/1eb92cce4cb993441dc91338e2220af0972d538e/.goreleaser.yml#L14) with `CGO_ENABLED=0`

**Comment by alex-dna-tech** [comment]
Fix applied. 
It can be closed as #2049.
